manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+PackageLocale: en-US
+Publisher: VoiceAttack.com
+PackageName: VoiceAttack
+License: Copyright © 2008-2020 VoiceAttack.com
+ShortDescription: Voice-activated control for your PC games and apps.
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0

```

Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```